src/shared/ui/MovieDetail/MovieDetail.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen } from "@testing-library/react";
+import userEvent from '@testing-library/user-event'
+import { MovieDetail } from './MovieDetail';
+import { movieDetailData } from './MovieDetailMockData';
+
+describe("MovieDetail Component Functionality", () => {
+    test("Should render movie details", () => {
```

Comment:
let's add some more test cases

Reply:
this component only shows data and that`s why I added only one test case for checking existence of data